docs/pages/docs/config/lists.md

```diff
@@ -277,15 +277,15 @@ Using GraphQL, to query a list named `settings`, with `isSingleton` set, you can
 ```graphql
 query {
   # singular (null or an item)
-  settings {
-    seoTitle
-    seoDescription
+  seoConfiguration {
+    title
+    description
   }
 
   # plural (0 or 1 items)
-  settingsMany {
-    seoTitle
-    seoDescription
+  seoConfigurations {
+    title
+    description
```

Comment:
Not a blocking thing but showing the many query feels unnecessary. It's not like you would actually want to use it. A note like "hey, the many things still exist for reasons" would certainly be helpful but actually showing it feels like we're saying that there's a reason people would want it when there isn't